File: app/src/main/java/dev/mainhq/schedules/database/AppDatabase.java
package dev.mainhq.schedules.database;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import dev.mainhq.schedules.database.dao.RoutesDAO;
import dev.mainhq.schedules.database.dao.StopTimesDAO;
import dev.mainhq.schedules.database.dao.TripsDAO;
import dev.mainhq.schedules.database.entities.*;

@Database(entities = {Routes.class, Trips.class, StopTimes.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract RoutesDAO routesDao();
    public abstract TripsDAO tripsDao();
    public abstract StopTimesDAO stopTimesDao();
}

File: app/src/main/java/dev/mainhq/schedules/database/AppDatabase.kt
package dev.mainhq.schedules.database;

import androidx.room.Database;
import androidx.room.RoomDatabase;
import androidx.room.TypeConverters

import dev.mainhq.schedules.database.dao.*;
import dev.mainhq.schedules.database.entities.*;

@Database(entities = [Routes::class, Trips::class, StopsInfo::class,
    Stops::class, Calendar::class, Shapes::class, Forms::class], version = 1)
@TypeConverters(Converters::class)
abstract class AppDatabase : RoomDatabase() {
    abstract fun routesDao() : RoutesDAO;
    abstract fun tripsDao() : TripsDAO;
    abstract fun stopsInfoDao() : StopsInfoDAO;
    abstract fun stopDao() : StopsDAO
    abstract fun calendarDao() : CalendarDAO
    abstract fun shapesDao() : ShapesDAO
    abstract fun formsDao() : FormsDAO
}
